--- gcc-4.7.2-ori/./libgcc/config/darwin-crt-tm.c	2012-02-16 10:46:31.000000000 +0100
+++ gcc-4.7.2/./libgcc/config/darwin-crt-tm.c	2013-04-29 15:05:29.000000000 +0200
@@ -26,8 +26,14 @@
 #include "tsystem.h"
 #include <stddef.h>
 #include <dlfcn.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+/*
 #include <mach-o/dyld.h>
 #include <mach-o/getsect.h>
+*/
 
 #ifdef __LP64__
 #define GET_DATA_TMCT(mh,size) \
--- gcc-4.7.2-ori/./libgomp/Makefile.am	2012-02-27 14:51:50.000000000 +0100
+++ gcc-4.7.2/./libgomp/Makefile.am	2013-04-18 17:59:49.000000000 +0200
@@ -28,7 +28,7 @@
 
 libgomp.ver: $(top_srcdir)/libgomp.map
 	$(EGREP) -v '#(#| |$$)' $< | \
-	  $(PREPROCESS) -P -include config.h - > $@ || (rm -f $@ ; exit 1)
+	  $(PREPROCESS) -include config.h - > $@ || (rm -f $@ ; exit 1)
 
 if LIBGOMP_BUILD_VERSIONED_SHLIB_GNU
 libgomp_version_script = -Wl,--version-script,libgomp.ver
--- gcc-4.7.2-ori/./libgomp/Makefile.in	2012-09-20 09:23:55.000000000 +0200
+++ gcc-4.7.2/./libgomp/Makefile.in	2013-04-18 18:00:15.000000000 +0200
@@ -1071,7 +1071,7 @@
 
 @LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@libgomp.ver: $(top_srcdir)/libgomp.map
 @LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@	$(EGREP) -v '#(#| |$$)' $< | \
-@LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@	  $(PREPROCESS) -P -include config.h - > $@ || (rm -f $@ ; exit 1)
+@LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@	  $(PREPROCESS) -include config.h - > $@ || (rm -f $@ ; exit 1)
 @LIBGOMP_BUILD_VERSIONED_SHLIB_SUN_TRUE@@LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@libgomp.ver-sun : libgomp.ver \
 @LIBGOMP_BUILD_VERSIONED_SHLIB_SUN_TRUE@@LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@		$(top_srcdir)/../contrib/make_sunver.pl \
 @LIBGOMP_BUILD_VERSIONED_SHLIB_SUN_TRUE@@LIBGOMP_BUILD_VERSIONED_SHLIB_TRUE@		$(libgomp_la_OBJECTS) $(libgomp_la_LIBADD)
